docs/zh/latest/FAQ.md

```diff
@@ -58,7 +58,7 @@ APISIX 是当前性能最好的 API 网关，单核 QPS 达到 2.3 万，平均
 4. 变化通知
 5. 高性能
 
-APISIX 需要一个配置中心，上面提到的很多功能是传统关系型数据库和KV数据库是无法提供的。与 etcd 同类软件还有 Consul、ZooKeeper等，更详细比较可以参考这里：[etcd why](https://github.com/etcd-io/etcd/blob/master/Documentation/learning/why.md#comparison-chart)，在将来也许会支持其他配置存储方案。
+APISIX 需要一个配置中心，上面提到的很多功能是传统关系型数据库和 KV 数据库是无法提供的。与 etcd 同类软件还有 Consul、ZooKeeper 等，更详细比较可以参考这里：[etcd why](https://etcd.io/docs/v3.3.12/learning/why/)，在将来也许会支持其他配置存储方案。
```

Comment:
We don't use v3.3

Reply:
Please provide a correct link.

Reply:
You could try `https://etcd.io/docs/v3.4.0/learning/why/` or `https://etcd.io/docs/current/learning/why/`

Reply:
updated